tests/load-tests/run.sh

```diff
@@ -35,6 +35,7 @@ else
         KUBE_SCHEDULER_LOG_PID=$!
     fi
     ## Run the actual load test
+    ## To enable progress bar , add `--enable-progress-bars` in [OPTIONS]
```

Comment:
Hello! I think what was meant was to print this message to the user when it is running the test, but this works for me as well.

Reply:
It is printed from within loadTests.go